WordPress/gutenberg
# Components: Add `info` prop support to MenuItem, FeatureToggle

Closes #10340 

This pull request seeks to add a new `info` prop to the `MenuItem` and `FeatureToggle` components.

![image](https://user-images.githubusercontent.com/1779930/47227679-6dc2c600-d391-11e8-9901-79b3cda69abd.png)

**Testing Instructions:**

Verify an extended description is shown for the More Menu Unified Toolbar button, as in the screenshot above:

Notable variations:

- Checked, unchecked
- Cross-browser

Ensure unit tests pass:

```
npm run test-unit
```
